Cargo.toml

```diff
@@ -14,6 +14,7 @@ keywords = ["trie", "patricia", "collection", "generic", "prefix"]
 categories = ["data-structures", "text-processing"]
 
 [dependencies]
+libc = "0.2"
```

Comment:
Just a small thing, but it would be good if this was an optional dependency behind a feature flag, see: https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section

Reply:
What feature name would you suggest?

Reply:
`ffi` or `c-bindings` or `c_bindings` would probably be good